app/models/country_petition_journal.rb

```diff
@@ -40,6 +42,7 @@ def self.reset!
         timezone('utc', now()), timezone('utc', now())
       FROM signatures
       WHERE state = 'validated'
+      AND country <> ''
```

Comment:
I guess I was playing safe, our schema allows for null `country` and I'm sure I read that in PSQL `<>` will act as both "NOT NULL" and "disallow blanks".  Could drop the condition though if we felt it was too much.